Cargo.toml

```diff
@@ -33,4 +35,11 @@ tonic-build = "0.6.0"
 # Workaround for https://github.com/hyperium/tonic/issues/243
 h2 = { git = "https://github.com/teozkr/h2.git", branch = "feature/grpc-uds" }
 
+kube = { git = "https://github.com/teozkr/kube-rs.git", branch = "feature/crd-complex-enum" }
```

Comment:
I think this can be removed now?

Reply:
Yes, once we upgrade to operator-rs 0.8.0.